src/css/Navbar.css

```
@@ -0,0 +1,191 @@
+}
+
+.navbar-dark .navbar-nav .nav-link:hover,
+.navbar-dark .navbar-nav .nav-link.active {
+    color: #ffaa4a;
+}
+
+.dropdown-menu {
+    background-color: #111;
+    border-top-color: #ffaa4a;
+    border-width: 2px;
+    border: radius 0;
```

Biome (1.9.4)

[error] Unexpected shorthand property border after border-top-color



(lint/suspicious/noShorthandPropertyOverrides)